MapboxNavigation/PollyVoiceController.swift

```diff
@@ -149,23 +158,31 @@ public class PollyVoiceController: RouteVoiceController {
                 return
             }
             
-            guard let data = data else { return }
+            guard let data = data else {
+                strongSelf.callSuperSpeak(strongSelf.fallbackText, error: "No data")
+                return
+            }
             
-            DispatchQueue.main.async {
-                do {
-                    strongSelf.audioPlayer = try AVAudioPlayer(data: data)
-                    strongSelf.audioPlayer?.delegate = self
+            do {
+                strongSelf.audioPlayer = try AVAudioPlayer(data: data)
+                let prepared = strongSelf.audioPlayer?.prepareToPlay()
+                
+                guard let _ = prepared else {
```

Comment:
If `prepared == false`, will this guard fall through?
I think we should unwrap `preparedToPlay() ?? false` so that the guard checks if prepared is true.

Reply:
I think if prepared is equal to false, it would get caught in this guard. But i can change it if you'd like.